projects/uitdatabank/docs/search-api/facets.md

```diff
@@ -0,0 +1,260 @@
+# Facets
+
+It is possible to get facet counts for specific fields using the `facets` URL parameter.
```

Comment:
I added an introduction text now, but I found it not easy to explain. I tried to focus on: 
- what is conceptually
- what can an integrator expect in the data
- what is used for

Reply:
I think it's good! I looked up a definition of faceted search on Wikipedia (because they usually try to explain concepts in an understandable manner), and I like the part where they say "allowing users to narrow down search results by applying multiple filters based on [faceted classification](https://en.wikipedia.org/wiki/Faceted_classification) of the items". https://en.wikipedia.org/wiki/Faceted_search

So maybe we could also incorporate that in some way.

And/or we could include a screenshot of an example of the faceted search in Widgets?

But overall this is already a very helpful explanation of the concept and we can always merge it like this and add the screenshot etc later.